CI/android/signing.properties

```diff
@@ -0,0 +1,4 @@
+STORE_FILE=vcmi-travis.jks
```

Comment:
Is this some sort of private signing key or just some placeholder?

Reply:
it's a real file which is public in the android repo. @nullkiller do you have more info?

Reply:
This is a keystore, it is password protected. And not the one used to sign releases

Reply:
I think it is used to sign dev builds from CI. Password is stored in github secret

Reply:
Interesting now how it works, are our new builds signed?

Reply:
daily builds are signed as before, the process is preserved

Reply:
Interesting how it extracts the secret

Reply:
Or maybe it is not protected properly, I never investigated it precisely.

Reply:
Looks like the password is hardcoded somewhere in CI/gradle scripts or there is none specified at all)

Reply:
Exactly, in this file. Not sure if we need to do anything about it so.